### PR TITLE
[SDPA-2709] Adds create ripple app scaffold tool.

### PR DESCRIPTION
**Pull request by dylankelly**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-2709

### Changed

1.  Adds tool to scaffold a new ripple nuxt site. See [README](https://github.com/dpc-sdp/ripple/pull/446/files#diff-910420569486f1bd6e66c12b84fe405d) for usage

### Screenshots

create-ripple-app

### CODE REVIEW

**Comment by dylankelly** [comment]
@tim-yao @alan-cole Can I get a CR on this one? Would be good if I could get some QA in creating a new project too, see Readme for usage

File: packages/create-ripple-app/template/_example/tide/modules/event/formdata.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
+export default {
```
Comment (tim-yao) [inline packages/create-ripple-app/template/_example/tide/modules/event/formdata.js]: What is this `_example` folder for?

File: packages/create-ripple-app/template/_package.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    "test:e2e": "cross-env NODE_ENV=test start-server-and-test start:build http://localhost:3000 cy:run",
+    <%_ } _%>
+    "lint": "eslint --ext .js,.vue . --max-warnings 0 && sass-lint",
+    "start": "run-script-os",
+    "start:default": "if [ \"$NUXT_HOT_RELOADING\" = \"1\" ] ; then nuxt; else nuxt start; fi",
+    "start:win32": "IF \"%NUXT_HOT_RELOADING%\" EQU \"1\" ( nuxt ) ELSE ( nuxt start )",
+    "start:build": "nuxt build && nuxt start"
+  },
+  "dependencies": {
+    "@dpc-sdp/ripple-nuxt-tide": "1.0.0-beta.10",
```
Comment (tim-yao) [inline packages/create-ripple-app/template/_package.json]: We may change this to `^1.0.0` later.

File: packages/create-ripple-app/template/_tests/test/e2e/fixtures/Forms/Grant/tc9c.csv [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿key,value,random
```
Comment (tim-yao) [inline packages/create-ripple-app/template/_tests/test/e2e/fixtures/Forms/Grant/tc9c.csv]: We may need a solution for updating these tests if they are added to custom site. Also, grants mainly only in vic gov au, do we need to add it to custom site?

File: packages/create-ripple-app/template/gitignore [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+.nuxt
+dist
+coverage
+
+# Dependencies lock files.
+package-lock.json
+
+# Do not ignore root and Ripple lock files.
+!/package-lock.json
+!/ripple/package-lock.json
```
Comment (tim-yao) [inline packages/create-ripple-app/template/gitignore]: Are we need these lines?

File: packages/create-ripple-app/template/_example/tide/modules/event/formdata.js (hunk 1 above)
Comment (dylankelly) [reply packages/create-ripple-app/template/_example/tide/modules/event/formdata.js]: Setup a new project with an example of how to extend the config. Useful for external agency clients, but we wouldn't want to add it to internal sites.

File: packages/create-ripple-app/template/_tests/test/e2e/fixtures/Forms/Grant/tc9c.csv (hunk 3 above)
Comment (dylankelly) [reply packages/create-ripple-app/template/_tests/test/e2e/fixtures/Forms/Grant/tc9c.csv]: True, I'm going to add a feature to only install the tests for the modules you have picked in another PR.

File: packages/create-ripple-app/template/gitignore (hunk 4 above)
Comment (dylankelly) [reply packages/create-ripple-app/template/gitignore]: nope, deleted

**Pull request merged**